File: test_programs/long.js
{
    const start = process.hrtime();
    var n = 0;
    for (var i = 0; i < 100000; i = i + 1) {
        for (var j = 0; j < i; j = j + 1) {
            n = n + j;
        }
    }

    const end = process.hrtime();

    console.log(n);
    console.log(end[0] - start[0] + end[1] / 1000000000 - start[1] / 1000000000);

}

File: test_programs/long.c
#include <time.h>
#include <stdio.h>

int main()
{
    clock_t start = clock();
    double n = 0;
    for (double i = 0; i < 100000; i = i + 1) {
        for (double j = 0; j < i; j = j + 1) {
            n = n + j;
        }
    }

    printf("%f\n", ((double)clock() - start) / CLOCKS_PER_SEC);
    printf("%f\n", n);
}
